pinot-core/src/test/java/org/apache/pinot/queries/TextSearchQueriesTest.java

```diff
@@ -1199,6 +1199,26 @@ public void testLuceneRealtimeWithSearcherManager()
     indexWriter.close();
   }
 
+  @Test
+  public void testLikeOperator()
```

Comment:
I don't think we should put the test under this class because `regexpLike` won't leverage text index currently